### DISCUSSION TITLE
Creating a support forum

### DISCUSSION THREAD

**Discussion by banesullivan** [pyvista/pyvista#2204] [Ideas]
Should we create a discourse forum to archive general use questions for PyVista?

I'm imagining something similar to VTK's: https://discourse.vtk.org

This would create a way to archive user questions and provide a place for new users to seek out community based help.

The slack channel has been filling this role, but it's not capable of being a support forum where questions can be searched

**Comment by akaszynski** [comment]
Absolutely!  I like the discourse format, and think it would be better than the slack.

**Comment by banesullivan** [comment]
Welp - I thought discourse was free... looks like we either pay a maintenance fee or set up our own server to host the site.

Anyone know of free alternatives?

**Comment by banesullivan** [comment]
I’m thinking the **best free alternative might be to create a separate repository** called `pyvista-support` under @pyvista where people can create issues for their general-use questions about the software. 

A benefit of this would be using GitHub’s new “transfer issue” feature which would help us move improperly raised issues to their proper places - e.g. if someone opens a general-use issue here, we can move it to `pyvista-support` and if an issue is raised in `pyvista-support` that describes a bug or feature-request, we could move it here.

Also, GitHub issues already have excellent markdown, code formatting, and file uploading support already built-in.

One thing to consider is that the scope of this support forum might become much bigger than @pyvista as I might encourage folks using `omfvista` to raise issues there and people using packages where `pyvista` is a supporting module might post questions that go beyond the scope of just `pyvista` but using `pyvista` with that other library

**Comment by banesullivan** [comment]
We need to add an issue template and screenshots on how to create a new issue

**Comment by akaszynski** [comment]
Agreed.  It would clean things up a bit on the main repo as well.  GitHub has great formatting and we could get people motivated to create a GitHub username and start contributing.
